js/src/wp-seo-dashboard-widget.js

```diff
@@ -97,7 +97,12 @@ class DashboardWidget extends React.Component {
 	 */
 	getFeed() {
 		// Developer note: this link should -not- be converted to a shortlink.
-		getPostFeed( "https://yoast.com/feed/widget/", 2 )
+		getPostFeed(
+			"https://yoast.com/feed/widget/?wp_version=" + wpseoDashboardWidgetL10n.wp_version +
+			"&php_version=" + wpseoDashboardWidgetL10n.php_version +
+			"&classic=" + wpseoDashboardWidgetL10n.classic,
```

Comment:
Please note that this output doesn't exactly match the instructions from the PR, where it said that the classic editor status (`0` or `1`) would appear directly behind the WP version, separated with a dash (`-`). For the functionality this doesn't matter, I think this is clearer actually :).

Reply:
Whoops, forgot to push the final changes while I did update the description :-)